trips/views.py

```diff
@@ -13,11 +13,15 @@
 def trips(user):
     try:
         user = User.objects.get(slug=user)
+        community = user.allow_community
     except User.DoesNotExist:
         abort(404)
-    return render_template('trips/map.html',
+    if community:
+        return render_template('trips/map.html',
                            user=user,
                            locations=user.locations)
+    else:
+        return render_template('private.html', user=user)
```

Comment:
`return render_template('private.html', user=user), 403`

This is the status code for _Forbidden_